test/e2e/specs/delete-prohibited/recordset/individual-facet.spec.ts

```diff
@@ -1,18 +1,22 @@
-import { test, expect } from '@playwright/test';
-
-// locators
-import RecordsetLocators from '@isrd-isi-edu/chaise/test/e2e/locators/recordset';
+import { test } from '@playwright/test';
 
 // utils
 import { getCatalogID } from '@isrd-isi-edu/chaise/test/e2e/utils/catalog-utils';
 import {
-  clearRangeInput, fillRangeInput, openFacet, openFacetAndTestFilterOptions,
-  openRecordsetAndResetFacetState, testClearAllFilters, testDefaultRangePickerInitialValues,
-  testRangeInputSubmitThenClear, testSelectFacetOption,
-  testSelectFacetOptionThenClear, testTimestampRangePickerInitialValues
+  openRecordsetAndResetFacetState, testIndividualFacet, TestIndividualFacetParams,
 } from '@isrd-isi-edu/chaise/test/e2e/utils/recordset-utils';
 
-const testParams: any = {
+const testParams: {
```

Comment:
I've avoided typing the `testParams` in specs that I've been writing for different reasons. We should make sure each spec is typed before we finalize the test migration